forage-android/src/main/java/com/joinforage/forage/android/ui/ForageElement.kt

```diff
@@ -5,6 +5,15 @@ import com.joinforage.forage.android.core.element.SimpleElementListener
 import com.joinforage.forage.android.core.element.StatefulElementListener
 import com.joinforage.forage.android.core.element.state.ElementState
 
+data class ForageContext(
+    val merchantId: String,
+    val sessionToken: String
+    // TODO: thoughts on customerId being part of the
```

Comment:
Let me chat with Rob about this, but I think we are going to want to include customerID in further requests. It may be smart to include it here. I'd say continue without adding it for now and I'll come back to this and update when I have more info.

Reply:
I was wrong! We don't need to include `customerId` here. It will only be used in the tokenize endpoint for the foreseeable future.

Reply:
OK sound good! Not an issue if we ever want to bring this back in the future too since it won't be a breaking change!

Reply:
removed the comment in [ff8a5be](https://github.com/teamforage/forage-android-sdk/pull/97/commits/ff8a5be12f0c3553abaad3ccf24280c7fb8c1eb1). Marking as resolved